bitnami/cert-manager/templates/controller/deployment.yaml

```
@@ -0,0 +1,135 @@
+apiVersion: {{ include "common.capabilities.deployment.apiVersion" . }}
+kind: Deployment
+metadata:
+  name: {{ include "common.names.fullname" . }}
```

Add the namespace (this apply to every manifest):
```suggestion
  name: {{ include "common.names.fullname" . }}
  namespace: {{ .Release.Namespace | quote }}
```